Editing the Introduction Section (#45)
greenelab/text_mined_hetnet_manuscript: content/02.introduction.md

## Introduction

Knowledge bases are important resources that hold complex structured and unstructured information. 
These resources have been used in important tasks such as network analysis for drug repurposing discovery [@doi:10.1371/journal.pone.0084912; @doi:10.1101/385617; @doi:10.7554/eLife.26726] or as a source of training labels for text mining systems [@doi:10.3115/1690219.1690287; @doi:10.1101/444398; @doi:10.1186/s12859-019-2873-7]. 
Populating knowledge bases often requires highly-trained scientists to read biomedical literature and summarize the results [@doi:10.1093/bib/bbn043].

> This process is referred to as manual curation, which requires a significant amount of effort and time.

Reviewer: ```suggestion
This time consuming process is referred to as manual curation.
```

In 2007 researchers estimated that filling a knowledge base via manual curation would require approximately 8.4 years to complete [@doi:10.1093/bioinformatics/btm229]. 

> As the rate of publications continues to exponentially increase [@doi:10.1002/asi.23329], using only manual curation becomes considerably impractical.  

Reviewer: ```suggestion
The rate of publications continues to exponentially increase [@doi:10.1002/asi.23329], so using only manual curation to fully populate a knowledge base has become impractical.  
```

> Relationship extraction has been studied as a solution towards handling the data overload problem [@doi:10.1093/bib/bbn043].

Reviewer: ```suggestion
Relationship extraction has been studied as a solution towards handling the challenge posed by an exponentially growing body of literature [@doi:10.1093/bib/bbn043].
```

This process consists of creating an expert system to automatically scan, detect and extract relationships from textual sources.

> Typically, these systems utilize machine learning techniques that often require large corpora of well-labeled training data.

Reviewer: ```suggestion
Typically, these systems utilize machine learning techniques that require large corpora of well-labeled training data.
```

> These datasets are difficult to obtain, because they are constructed via some form of manual curation.

Reviewer: ```suggestion
These corpora are difficult to obtain, because they are constructed via particularly detailed manual curation.
```

> Distant supervision is a technique used to sidestep the dependence of manual curation and quickly generates large datasets.

Reviewer: ```suggestion
Distant supervision is a technique designed to sidestep the dependence on manual curation and quickly generate large training datasets.
```

> This technique makes the assumption that positive examples have already been established in selected databases [@doi:10.3115/1690219.1690287].

Reviewer: ```suggestion
This technique makes the assumption that positive examples established in selected databases can be applied to any sentence that contains them [@doi:10.3115/1690219.1690287].
```
